packages/ckeditor5-dev-release-tools/tests/tasks/commitandtag.js

```diff
@@ -169,12 +170,47 @@ describe( 'commitAndTag()', () => {
 		} ) ).rejects.toThrow( 'Error executing git commit in dry run mode.' );
 
 		expect( stubs.git.log ).toHaveBeenCalledExactlyOnceWith( [ '-1' ] );
-		expect( stubs.git.commit ).toHaveBeenCalledExactlyOnceWith( 'Release: v1.0.0. [skip ci]', [
+		expect( stubs.git.add ).toHaveBeenCalledExactlyOnceWith( [
 			'package.json',
 			'packages/ckeditor5-foo/package.json'
 		] );
+		expect( stubs.git.commit ).toHaveBeenCalledExactlyOnceWith( 'Release: v1.0.0. [skip ci]' );
 
 		expect( stubs.git.reset ).not.toHaveBeenCalled();
 		expect( stubs.git.addAnnotatedTag ).not.toHaveBeenCalled();
 	} );
+
+	it( 'should split adding files to commit into chunks if they exceed the character limit', async () => {
+		const numberOfFiles = 500;
+
+		vi.mocked( glob ).mockResolvedValue(
+			new Array( numberOfFiles ).fill().map( ( _, i ) => `very/long/path/foo/bar/file_${ i }.js` )
+		);
+
+		await commitAndTag( { version: '1.0.0', packagesDirectory: 'packages', files: [ '**/package.json' ] } );
+
+		expect( vi.mocked( glob ) ).toHaveBeenCalledExactlyOnceWith( expect.anything(), expect.objectContaining( {
+			absolute: true,
+			nodir: true
+		} ) );
```

Comment:
This assertion is not needed in this unit test, because (as the test name suggests), it verifies only if files are split into chunks.